dev/templates/block/editor.pcss.mustache

```diff
@@ -0,0 +1,9 @@
+/* -------------------------------------------------------------------------
+ *
+ * Block - {{title}} - Styles - Editor Only
+ *
+ * ------------------------------------------------------------------------- */
+
+{{#isDynamicVariant}}section{{/isDynamicVariant}}.wp-block-{{namespace}}-{{slug}} {
```

Comment:
Just a note here that this prefixed `section` is due to dynamic blocks rendering the wrapper div + the wrapping `section` in the render template (`render.php`). We're only wanting to target the section (the actual block) in this case.